src/main/java/com/powsybl/openloadflow/network/impl/LfNetworkLoaderImpl.java

```diff
@@ -503,6 +511,65 @@ private static void createBranches(List<LfBus> lfBuses, LfNetwork lfNetwork, LfT
         }
     }
 
+    private static void updateControlArea(Bus bus, LfBus lfBus, LfNetwork network, LfNetworkParameters parameters, LoadingContext loadingContext, LfNetworkLoadingReport report) {
+        Optional<Area> areaOpt = bus.getVoltageLevel().getArea(parameters.getAreaInterchangeControlAreaType());
```

Comment:
You should explain here that only areas necessary to the AIC are created in the LfNetwork

Reply:
changed in 5a29ee99ce070c764b03988fb43c2636eb909798